lib/hammer_cli_foreman_discovery/discovery.rb

```diff
@@ -83,6 +90,21 @@ def request_params
       build_options
     end
 
+    class AutoProvisionCommand < HammerCLIForeman::SingleResourceCommand
+      action :auto_provision
+      command_name "auto_provision"
```

Comment:
I just noticed that we have the same problem with `refresh_facts` from https://github.com/theforeman/hammer-cli-foreman-discovery/pull/2 I overlooked it in the review.